src/EventSubscriber/CreateNewWishlistSubscriber.php

```diff
@@ -81,25 +56,30 @@ public function onKernelRequest(RequestEvent $event): void
             return;
         }
 
+        $request = $this->requestStack->getMainRequest();
```

Comment:
Maybe it would be a good idea to check if the request isn't null? Because the method looks like this: public function getMainRequest(): ?Request.

Reply:
The idea is to use it only in HTTP mode, so I can do assertions for not null :)